CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1228,10 +1228,15 @@ end
 local function OnCoreGuiChanged(coreGuiType, enabled)
 	-- Check for enabling/disabling the whole thing
 	if coreGuiType == Enum.CoreGuiType.Backpack or coreGuiType == Enum.CoreGuiType.All then
-		enabled = enabled and (TopbarEnabled or UserInputService.VREnabled)
+		enabled = enabled and TopbarEnabled 
 		WholeThingEnabled = enabled
 		MainFrame.Visible = enabled
 
+		if UserInputService.VREnabled then
+			local Panel3D = require(RobloxGui.Modules.VR.Panel3D)
+			Panel3D.Get("Backpack"):SetVisible(enabled)
```

Comment:
We set BackpackPanel to invisible when we open a module in VR:
` 			VRModuleOpenedConn = VRHub.ModuleOpened.Event:connect(function(moduleName)`
`				local openedModule = VRHub:GetModule(moduleName)`
`				if openedModule ~= BackpackScript and openedModule.VRIsExclusive then`
`					BackpackPanel:SetVisible(false)`

I think we should keep track of the variables controlling the visibility of Backpack and whenever we SetVisible it we should consult all those variables.